package.json

```diff
@@ -197,7 +197,7 @@
     "stylelint-processor-styled-components": "^1.10.0",
     "svg-sprite": "^2.0.2",
     "ts-node": "^10.9.1",
-    "typescript": "^4.8.4",
+    "typescript": "^5.1.0",
```

Comment:
What's the rationale for going to 5.10, as 5.1.6 is recommended and 5.2.2 is latest?

Reply:
Looks like 5.1.6 should be compatible.